### PR TITLE
Liam Coles HW 1

### PR DESCRIPTION
**Pull request by liamcoles**
I wrote this locally in R studio. Then I copied it into Github. The major difficulties that I had with this homework were mainly operating github. I have problems figuring out exactly how several of the functions were, especially with posting onto the master branch. Hopefully, I don't submit this homework several times accidentally. Additionally, for my bio, I wanted to upload a picture, but I was unable to retrieve it from my images folder. Hopefully, I will remedy this issue in the coming week.

### CODE REVIEW

**Comment by cysakamoto** [comment]
Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |    x       |              |            |
| **Coding strategy**         |     x      |              |            |
| **Presentation**            |      x     |              |            |
| **Achievement, creativity** |           |     x         |            |
| **Ease of access**          |    x       |              |            |

Hi Liam, 

Great page, I like how you experimented with different markdowns too. I missed an image for your page, but otherwise, great work! 

Cristina

File: README.md [hunk 1]
```diff
@@ -1,2 +1,28 @@
 # hw01
 Repository for homework 01
+
+## Liam Coles's first homework (The double hashtag gives me section title)
+
+My name is Liam Coles. I am a BA/MA student. I am majoring in Econ but doing my MA in psychology. I am from New York and really like philosophy. I am excited to learn R because it is helpful for my research. 
+
+*an asterisk makes this text italic*
```
Comment (smiklin) [inline README.md:8]: I really like how you made this code into a bit of a cheat-sheet!

File: README.md [hunk 2]
```diff
@@ -1,2 +1,28 @@
...
+
+*an asterisk makes this text italic* 
+
+_so will an underscore_
+
+**doubling either makes it bold**
+
+One of my favorite websites is http://www.fivethirtyeight.com/
+
+One of my favorite quotes is from Aristotle's Metaphysics, Book Lamda:
```
Comment (smiklin) [inline README.md:16]: Nice quote!

**Review by smiklin — COMMENTED** [review]
Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |      v     |              |            |
| **Coding strategy**         |   v        |              |            |
| **Presentation**            |      v     |              |            |
| **Achievement, creativity** |      v     |              |            |
| **Ease of access**          |     v      |              |            |

You seem to have experimented with all the formatting features really creatively!
I struggled with the picture as well, but I'm feeling that the easiest way to do it is to upload it and then use it's URL,  with the code `caption` ?

**Pull request closed without merging**